cookbook/images/network-image.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Display images from the internet"
+permalink: /cookbook/images/network-image/
+---
+
+Displaying images is fundamental for most mobile apps. Flutter provides the 
```

Comment:
I think there have been concerns about images just "popping in" after a network load. Or, wanting to show a placeholder.

Basically, the basic image widget usually isn't what people want, ultimately. It's a building block, yes, but I think we should list the shortcomings and then have recipes for:

* managing the image cache
* fading in an image
* placeholder image

We might want to mention fading in images and placeholder images.

For example: https://pub.dartlang.org/packages/cached_network_image

Reply:
Yep! I'm going to add recipes for these cases as well, but wanted to start with the building blocks and work up to more complex examples.